src/wavesurfer.js

```diff
@@ -149,6 +150,25 @@ import MediaElementWebAudio from './mediaelement-webaudio';
  * the dependency specified in extends. Returns the plugin class.
  */
 
+/**
+ * @typedef {Object} SplitChannelOptions
+ * @desc parameters applied when splitChannels option is true
+ * @property {boolean} overlay=false determines whether channels are rendered on top of each other or on separate tracks
+ * @property {object} channelColors={} object describing color for each channel. Example:
+ * {
+ *     0: {
+ *         progressColor: 'green',
+ *         waveColor: 'pink'
+ *     },
+ *     1: {
+ *         progressColor: 'orange',
+ *         waveColor: 'purple'
+ *     }
+ * }
+ * @property {number[]} filterChannels=[] indexes of channels to be hidden from rendering
+ * @property {boolean} relativeNormalization=false determines whether normalization is done per channel or maintains proportionality between channels. Only applied when normalize and splitChannels are both true.
```

Comment:
add version marker when this was introduced

Reply:
I'm sorry, I'm not sure I understand how to do that. I am checking out esdoc and jsdoc documentation for references for something similar but I haven't found anything that seems pertinent yet. Do you have an example I could look at for format?